style.css

```diff
@@ -0,0 +1,128 @@
+body {
+	margin: 0;
+}
+
+#navbar {
+	position: fixed;
+	top: 0px;
+	width: 100%;
+
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	background: lightsalmon;
+}
+
+#navbar a {
+	text-decoration: none;
+	display: block;
+	padding: 1em;
+	font-size: 1.5em;
+	font-weight: bold;
+	color: white;
+}
+
+#navbar a:hover {
+	background: salmon;
+}
+
+main {
+	display: grid;
+	grid-template-rows: 100vh auto 300px 100px;
+}
+
+#welcome-section {
+	text-align: center;
+	background: #fdf9c4;
+	padding-top: 45vh;
+}
+
+#projects {
+	text-align: center;
+	background: white;
+}
+
+#profile {
+	text-align: center;
+	background: #fdf9c4;
+	display: grid;
+	grid-template-columns: 1fr;
+	justify-items: center;
+}
+
+.circle {
+	background: #ff6961;
+	border-radius: 50%;
+	width: 100px;
+	height: 100px;
+	text-align: center;
+	vertical-align: middle;
+	line-height: 100px;
+}
+
+#profile-link {
+	text-decoration: none;
+	color: black;
+	font-size: 1.2em;
+}
+
+#copyright {
+	text-align: center;
+	background: lightsalmon;
+	padding: 1em;
+}
+
+.tiles {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr 1fr;
+	justify-items: center;
+}
+
+.project-tile {
+	background: white;
+	padding: 2em;
+}
+
+.project-tile a {
+	text-decoration: none;
+	color: #ff6961;
+	font-weight: bold;
+	font-size: 0.9em;
+}
+
+.responsive {
+	max-width: 100%;
+	height: auto;
+}
+
+@media all and (max-width: 800px) {
+	#navbar {
+		justify-content: space-around;
+	}
+
+	.tiles {
+		grid-template-columns: 1fr 1fr;
+	}
+}
+
+@media all and (max-width: 600px) {
+	#navbar {
+		flex-flow: column wrap;
+		padding: 0;
+	}
+
+	#navbar a {
+		text-align: center;
+		padding: 10px;
+		border-top: 1px solid rgba(255, 255, 255, 0.3);
+		border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+	}
+
+	#navbar a:last-of-type {
+		border-bottom: none;
+	}
+
+	.tiles {
+		grid-template-columns: 1fr;
+	}
+}
```

Comment:
- "Kindly I recommend you to remove those empty spaces to follow the best practice of CSS practice with the correct indentation. "